docs/user/changestack.md

```diff
@@ -3,7 +3,7 @@
 
 Notably the Version of Node-RED which a project uses, you can now change the stack of an existing project which allows you to upgrade the version of Node-RED
 
-**Note:** An administrator must make a specific version of Node-RED availble by creating the relevent stack
+**Note:** An administrator must make a specific version of Node-RED available by creating the relevant stack
```

Comment:
I wonder if there's a GitHub action that can spell check `.md` files?

Reply:
https://github.com/marketplace/actions/run-pyspelling-as-a-github-action looks like a candidate.

I have one in VSCode but its not very obvious